CleverCloud/clever-components
# cc-article-list: fix XML parser

Fixes #867 

## How to review?

* Check the UI
  * Go to `https://clever-components-preview.cellar-c2.services.clever-cloud.com/fix-article-smart/index.html?path=%2Fstory%2F%F0%9F%9B%A0-homepage-cc-article-list-%F0%9F%92%A1-smart--page` 
  * Check that the component load correctly
  * Check both english and french
  * (With the new RSS the description is longer, I can't really do much about that sadly)
* Review the code except the sandbox commit. I thought we could use it but the smart page can act as a sandbox for us :wink:
